### PR TITLE
Problem:variable 'swapped' set but not used

### PR DESCRIPTION
**Pull request by Riddance**
fixed error: variable 'swapped' set but not used

### CODE REVIEW

**Comment by Riddance** [comment]
fixed error: variable 'swapped' set but not used

File: src/zlist.c [hunk 1]
```diff
@@ -394,14 +394,14 @@ zlist_sort (zlist_t *self, zlist_compare_fn *compare)
     bool swapped = false;
     while (gap > 1 || swapped) {
         if (gap > 1)
-          gap = (int) ((double) gap / 1.3);
+            gap = (int) ((double) gap / 1.3);
```
Comment (jemc) [inline src/zlist.c]: Can you revert this part of the change?  @hintjens just sent in [a PR to normalize the whitespace](https://github.com/zeromq/czmq/pull/990), and he shouldn't have to do it again for the same line within a few hours :wink:.

See http://rfc.zeromq.org/spec:21 for applicable style guidelines.

Comment (Riddance) [inline src/zlist.c]: sorry, i reverted it

**Comment by hintjens** [comment]
@Riddance thanks for fixing zlist_sort. This is one of my favorite methods. :-)

**Pull request merged**